### DISCUSSION TITLE
Deamon on openrc

### DISCUSSION THREAD

**Discussion by los95** [AdnanHodzic/auto-cpufreq#219] [Q&A]
Is it possible to start a service on openrc? I tried sudo rc-service auto-cpufreq start and it says that it doesn't exist.

**Comment by explosion-mental** [comment]
@los95 I'm pretty sure it can. I have no experience with openrc so I need help on how to identify if the system is using openrc. Output of:
- `readlink -f /sbin/init`
- `ps 1` and `ps h -o comm 1`

After that all we do is copy the service file to the corresponding directory. On runit service files are scripts, maybe on openrc as well?
